pkg/controller/vitesskeyspace/vitesskeyspace_controller.go

```diff
@@ -179,6 +184,19 @@ func (r *ReconcileVitessKeyspace) Reconcile(request reconcile.Request) (reconcil
 	topoResult, err := r.reconcileTopology(ctx, vtk)
 	resultBuilder.Merge(topoResult, err)
 
+	wr, err := newWrangler(ctx, vtk.Spec.GlobalLockserver)
+	if err != nil {
+		r.recorder.Eventf(vtk, corev1.EventTypeWarning, "StatusUpdateWarning", "failed to retrieve resharding information: %v", err)
+	}
+
+	workflows, err := wr.ListAllWorkflows(ctx, vtk.Spec.Name)
+	if err != nil {
+		r.recorder.Eventf(vtk, corev1.EventTypeWarning, "StatusUpdateWarning", "failed to list active resharding workflows: %v", err)
+		workflows = make([]string, 0)
+	}
+	vtk.Status.ReshardingInProgress = len(workflows) != 0
```

Comment:
That's a good point - but if that's the case I can't think of a way to know for sure if resharding is in progress without storing state about workflows we intend to execute. From an OSS perspestive, it might not even be possible at all.